src/modules/simulation/gz_bridge/GZBridge.cpp

```diff
@@ -148,7 +148,7 @@ int GZBridge::init()
 
 				// If Gazebo has not been called, wait 2 seconds and try again.
 				else {
-					PX4_WARN("Service call timed out as Gazebo has not been detected.");
+					printf("WARN [gz_bridge] Service call timed out as Gazebo has not been detected. \n");
```

Comment:
Why?

Reply:
There is a bug with the PX4 messaging system where text in colour (like the one generated by `PX4_WARN`) will not be written into a textfile. I can try to reproduce it and make a bug report.